_dashboards/discover/exploring-query-enhancements.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
@sejli Please tech review this documentation. It's the new content. All other files are changes to navigation and don't require tech approval. Thank you.